go/toolchains.rst

```diff
@@ -18,22 +27,568 @@ customize the behavior of the core_ go rules.
+| This must be the same go toolchain object you got this function from.                            |
++--------------------------------+-----------------------------+-----------------------------------+
+| :param:`sources`               | :type:`list<File>`          | |mandatory|                       |
++--------------------------------+-----------------------------+-----------------------------------+
+| An iterable of source code artifacts.                                                            |
++--------------------------------+-----------------------------+-----------------------------------+
+| :param:`importpath`            | :type:`string`              | :value:`""`                       |
++--------------------------------+-----------------------------+-----------------------------------+
+| The import path this package represents. This is passed to the -p flag.                          |
++--------------------------------+-----------------------------+-----------------------------------+
+| :param:`golibs`                | :type:`list<GoLibrary>`     | :value:`[]`                       |
++--------------------------------+-----------------------------+-----------------------------------+
+| An iterable of all imported libraries.                                                           |
++--------------------------------+-----------------------------+-----------------------------------+
+| :param:`mode`                  | :type:`string`              | :value:`NORMAL_MODE`              |
++--------------------------------+-----------------------------+-----------------------------------+
+| Controls the compilation setup affecting things like enabling profilers and sanitizers.          |
+| This must be one of the values in common.bzl#compile_modes.                                      |
+| See modes_ for more information.                                                                 |
++--------------------------------+-----------------------------+-----------------------------------+
+| :param:`out_lib`               | :type:`File`                | |mandatory|                       |
++--------------------------------+-----------------------------+-----------------------------------+
+| The archive file that should be produced.                                                        |
++--------------------------------+-----------------------------+-----------------------------------+
+| :param:`gc_goopts`             | :type:`string_list`         | :value:`[]`                       |
++--------------------------------+-----------------------------+-----------------------------------+
+| Additional flags to pass to the compiler.                                                        |
++--------------------------------+-----------------------------+-----------------------------------+
+
+
+cover
+~~~~~
+
+The compile function adds an action that runs ``go tool cover`` on a set of source files
```

Comment:
s/The compile function/The cover function/

Note that this removes most comments, including cgo comments.